sysdig/resource_sysdig_secure_cloud_auth_account_test.go

```diff
@@ -441,3 +441,66 @@ func TestAccAWSSecureCloudAccountFCCSPM(t *testing.T) {
 		},
 	})
 }
+
+func XTestGCPAgentlesScanningOnboarding(t *testing.T) {
```

Comment:
this test is not passing now, but on the original PR it did.
need some help here @cgeers  :)
any ideas? i'm guessing it has to do with the un/marshalling.. but not sure